core/src/ancestor_hashes_service.rs

```diff
@@ -278,12 +312,14 @@ impl AncestorHashesService {
             );
         }
         time.stop();
```

Comment:
Gross, just expanded the context and realized `time` is a `Measure` (which is for metrics) for no good reason. Mind switching it to an `Instant`?